.github/scripts/anaconda-prune/prune.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
```

Comment:
Please mention in PR description where this scrip is coming from, because if this is copy-n-paste from trunk, where it's already working one does not need to review this part

Reply:
Yes, I should have put those time-saving reminders in the PR description. It is done only now.